tasks/main.yml

```
@@ -67,6 +67,24 @@
         state: touch
       when: ansible_distribution == "Fedora"
 
+- name: selinux
+  block:
+
+  - name: Generate selinux policy for freshclam
+    ansible.builtin.shell: grep freshclam /var/log/audit/audit.log | audit2allow -M freshclam
+
+  - name: Apply selinux policy for freshclam
+    ansible.builtin.shell: semodule -i freshclam.pp
```

Prefer long form options for readability:
```suggestion
    ansible.builtin.shell: semodule --install freshclam.pp
```